### DISCUSSION TITLE
Microsoft Windows and Office 365

### DISCUSSION THREAD

**Discussion by OLLI-S** [microsoft/winget-pkgs#57901] [Ideas]
It would be cool to see Windows Updates and also updates of Office 365 in winget.
Then I have **ONE TOOL** that keeps all my apps and my system up-to date.

**Comment by ArminWeb** [comment]
Would love to see this in WinGet

**Comment by lychichem** [comment]
I suggest combining them to Windows update. there is an option in Windows update, that is, "offer updates to other programs". We can use it to control winget. Also, more users will like GUI rather than CUI.

**Reply by OLLI-S** [reply]
Fantastic Idea!
So we have a "one-click" solution for updates.
Great UX.

**Reply by OLLI-S** [reply]
@lychichem In my German version of Windows 10 I have the option "Allow Updates for additional **Microsoft** products" (translated this from the German UI, so the text might not be 1:1 to the US version of Windows 10.
Would be cool, if this also includes other products (not only MS products).

**Reply by lychichem** [reply]
> @lychichem In my German version of Windows 10 I have the option "Allow Updates for additional **Microsoft** products" (translated this from the German UI, so the text might not be 1:1 to the US version of Windows 10.
> Would be cool, if this also includes other products (not only MS products).

yes, now it is only my imagination. and i use Chinese, so there may have improper ttanslation.
